server/src/migration/1625857050805-ModifyObservation.ts

```diff
@@ -0,0 +1,18 @@
+import {MigrationInterface, QueryRunner} from 'typeorm';
```

Comment:
I need to see if there's an eslint rule for this...

```suggestion
import {
  MigrationInterface,
  QueryRunner,
} from 'typeorm';
```

Reply:
ok

Reply:
IMO, this is a terrible style for the 21st century with large HD to 4k screens...why split it on multiple lines making us to scroll more? But whatever works for you....

Reply:
I personally find it a very worthwhile tradeoff to scroll more in order to keep things readable and to keep diffs extra clear, while also making it easy to trim/comment things out. I think it's much easier to scan code and catch mistakes/changes when you don't have to read it like a long sentence on one line. Even though we have such wide screens nowadays, I don't think it's a good idea to try and stretch code too far across them.